docs/MSSPContentPackDoc.md

```diff
@@ -1,40 +0,0 @@
-# Multi-tenancy Incident Response Solution Pack v1.0.0
```

Comment:
Why we have removed all files from docs?

Reply:
we should need to update the doc .. that's why I removed it... Once updated I will add it again

Reply:
sounds good. Please make a note of it.